tests/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+document: test-metadata
+subtype: general
```

Comment:
Can you please add more comments here?

Reply:
hmm, we can add these comment inside container-image-template, here it is more less PoC, comments are not so imprtant, or if you would like know more, see https://github.com/fedora-modularity/meta-test-family/blob/devel/mtf/metadata/examples/general-component/tests/metadata.yaml

comments here will just duplicate comments inside this commented config

Reply:
I've added link to official commented example to know more